Fixed AttributeError: 'NoneType' object has no attribute 'group'

evandrocoan opened the pull request:
```
Traceback (most recent call last):
  File "D:\SublimeText\sublime_plugin.py", line 939, in run_
    return self.run(edit)
  File "D:\SublimeText\Data\Packages\LaTeX Word Count\WordCount.py", line 136, in run
    language = find_language(syntax).group(1)
```

evandrocoan committed:
Formatted the WordCount.py statistics message display.

evandrocoan committed:
Fixed AttributeError: 'NoneType' object has no attribute 'group'

Traceback (most recent call last):
  File "D:\SublimeText\sublime_plugin.py", line 939, in run_
    return self.run(edit)
  File "D:\SublimeText\Data\Packages\LaTeX Word Count\WordCount.py", line 136, in run
    language = find_language(syntax).group(1)

kevinstadler commented:
That looks great, could you just provide some more information on which OS, ST version and Python version you got the original AttributeError with (and under what circumstances)?

evandrocoan commented:
I am using Windows 10 build 15063, with Sublime Text build 3154, x86, with python 3.3.6

I just opened a new unsaved view with the syntax:
```
>>> view.settings().get("syntax")
'Packages/C++/C++.tmLanguage'
```
And I tried to count the words on the text, but this error showed up, then I opened this package code and fixed it.

evandrocoan committed:
Fixed TypeError: 'bool' object is not callable, line 151:

(rwords, rchars, rtotal) = wordcount_fn(self.view.substr(region))

evandrocoan committed:
Fixed the packages not replacing the default Sublime Text keys

Thread:
Is possible to package override a Sublime Text Default keybinding?
https://forum.sublimetext.com/t/is-possible-to-package-override-a-sublime-text-default-keybinding/31688

evandrocoan committed:
Created a .no-sublime-package for all packages, in attempt to fix

evandrocoan/ITE 108 - Expand Region keybind is not working on stable version.

evandrocoan committed:
Revert "Created a .no-sublime-package for all packages, in attempt to fix"

This reverts commit 26760bda0b9f19ec0e0a2f37075b14be53727bcc.

evandrocoan committed:
Merge pull request #1 from kevinstadler/master

Update from upstream repo kevinstadler/SublimeLaTeXWordCount@master

evandrocoan committed:
Fixed the usage of SublimeText instead of evandroforks on

.travis.yml and appveyor.yml

evandrocoan committed:
Created the codecov.yml with threshold: 2% to fix it

failing the build by small threshold changes
https://github.com/mozilla-mobile/firefox-tv/pull/779

evandrocoan committed:
Fixed the usage of evandroforks instead of evandrocoan on

.travis.yml and appveyor.ps1

evandrocoan committed:
Updated README.md installation instructions
